===================================================================
--- theirs.js
+++ ours.js
@@ -12,21 +12,23 @@
     render: withGraphQLQuery(
       "node(1234567890){image{uri}}",
       function (container, data) {
         return (
-          <div>
-            <InlineBlock>
-              <img
-                src={data[1234567890].image.uri}
-                style={{
-                  position: "absolute",
-                  top: "0",
-                  left: "0",
-                  zIndex: "-1",
-                }}
-              />
-            </InlineBlock>
-          </div>
+          (
+            <div>
+              <InlineBlock>
+                <img
+                  src={data[1234567890].image.uri}
+                  style={{
+                    position: "absolute",
+                    top: "0",
+                    left: "0",
+                    zIndex: "-1",
+                  }}
+                />
+              </InlineBlock>
+            </div>
+          )
         );
       },
     ),
   },
@@ -42,12 +44,14 @@
 
 (function webpackUniversalModuleDefinition() {})(
   this,
   function (__WEBPACK_EXTERNAL_MODULE_85__, __WEBPACK_EXTERNAL_MODULE_115__) {
-    return /******/ (function (modules) {
-      // webpackBootstrap
-      /******/
-    })(
+    return /******/ (
+      function (modules) {
+        // webpackBootstrap
+        /******/
+      }
+    )(
       /************************************************************************/
       /******/ [
         /* 0 */
         /***/ function (module, exports, __webpack_require__) {
