===================================================================
--- theirs.ts
+++ ours.ts
@@ -2,20 +2,13 @@
 
 type Constructor<T> = new (...args: any[]) => T;
 
 class Base {
-  constructor(
-    public x: number,
-    public y: number,
-  ) {}
+  constructor(x: number, y: number) {}
 }
 
 class Derived extends Base {
-  constructor(
-    x: number,
-    y: number,
-    public z: number,
-  ) {
+  constructor(x: number, y: number, z: number) {
     super(x, y);
   }
 }
 
