===================================================================
--- theirs.js
+++ ours.js
@@ -14,11 +14,13 @@
   3 +
   2 +
   3 +
   2 +
-  (function () {
-    return 3;
-  })() +
+  (
+    function () {
+      return 3;
+    }
+  )() +
   3 +
   2 +
   3 +
   2 +
@@ -38,13 +40,15 @@
     3 +
     2 +
     3 +
     2 +
-    (function () {
-      const x = 5;
+    (
+      function () {
+        const x = 5;
 
-      return x;
-    })() +
+        return x;
+      }
+    )() +
     3 +
     2 +
     3 +
     2 +
@@ -103,11 +107,13 @@
 
 // https://github.com/prettier/prettier/issues/1626#issue-229655106
 const Bar = styled.div`
   color: ${(props) =>
-    props.highlight.length > 0
-      ? palette(["text", "dark", "tertiary"])(props)
-      : palette(["text", "dark", "primary"])(props)} !important;
+    (
+      props.highlight.length > 0
+        ? palette(["text", "dark", "tertiary"])(props)
+        : palette(["text", "dark", "primary"])(props)
+    )} !important;
 `;
 
 // https://github.com/prettier/prettier/issues/3368
 let message = `this is a long message which contains an interpolation: ${format(data)} <- like this`;
