===================================================================
--- theirs.js
+++ ours.js
@@ -1,16 +1,17 @@
-x =
-  (bifornCringerMoshedPerplexSawder) =>
-  (askTrovenaBeenaDependsRowans, glimseGlyphsHazardNoopsTieTie) =>
-  (f00) => {
-    averredBathersBoxroomBuggyNurl();
-  }; // BOOM
+x = (bifornCringerMoshedPerplexSawder) =>
+  (
+    (askTrovenaBeenaDependsRowans, glimseGlyphsHazardNoopsTieTie) => (f00) => {
+      averredBathersBoxroomBuggyNurl();
+    } // BOOM
+  );
 
-x2 =
-  (a) =>
+x2 = (a) =>
   (
-    askTrovenaBeenaDependsRowans1,
-    askTrovenaBeenaDependsRowans2,
-    askTrovenaBeenaDependsRowans3,
-  ) => {
-    c();
-  } /* ! */; // KABOOM
+    (
+      askTrovenaBeenaDependsRowans1,
+      askTrovenaBeenaDependsRowans2,
+      askTrovenaBeenaDependsRowans3,
+    ) => {
+      c();
+    } /* ! */ // KABOOM
+  );
