===================================================================
--- theirs.js
+++ ours.js
@@ -60,13 +60,13 @@
 const create = () => {
   const result = doSomething();
   return (
     shouldReturn &&
-    result.ok && {
-      status: "ok",
-      createdAt: result.createdAt,
-      updatedAt: result.updatedAt,
-    }
+      result.ok && {
+        status: "ok",
+        createdAt: result.createdAt,
+        updatedAt: result.updatedAt,
+      }
   );
 };
 
 const create2 = () => {
