===================================================================
--- theirs.ts
+++ ours.ts
@@ -1,8 +1,8 @@
 // Tests that readonly parameter properties behave like regular readonly properties
 
 class A {
-  constructor(readonly x: number) {
+  constructor(x: number) {
     this.x = 0;
   }
 }
 
@@ -16,23 +16,23 @@
 
 class C extends A {
   // This is the usual behavior of readonly properties:
   // if one is redeclared in a base class, then it can be assigned to.
-  constructor(readonly x: number) {
+  constructor(x: number) {
     super(x);
     this.x = 1;
   }
 }
 
 class D {
-  constructor(private readonly x: number) {
+  constructor(x: number) {
     this.x = 0;
   }
 }
 
 // Fails, can't redeclare readonly property
 class E extends D {
-  constructor(readonly x: number) {
+  constructor(x: number) {
     super(x);
     this.x = 1;
   }
 }
