===================================================================
--- theirs.js
+++ ours.js
@@ -9,14 +9,16 @@
 });
 
 // Missing one level of indentation because of the comment
 const rootEpic = (actions, store) =>
-  combineEpics(...epics)(actions, store)
-    // Log errors and continue.
-    .catch((err, stream) => {
-      getLogger().error(err);
-      return stream;
-    });
+  (
+    combineEpics(...epics)(actions, store)
+      // Log errors and continue.
+      .catch((err, stream) => {
+        getLogger().error(err);
+        return stream;
+      })
+  );
 
 // optional trailing comma gets moved all the way to the beginning
 const regex = new RegExp(
   "^\\s*" + // beginning of the line
