===================================================================
--- theirs.ts
+++ ours.ts
@@ -1,50 +1,50 @@
 type Type = {
   // comment
-  readonly [T in number];
+  readonly [T in number]?;
 };
 
 type Type = {
   // comment1
   // comment2
-  readonly [T in number];
+  readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  +readonly [T in number];
+  +readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  -readonly [T in number];
+  -readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  +readonly [T in number];
+  +readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  +readonly [T in number];
+  +readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  readonly [T in number];
+  readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  [T in number];
+  readonly [T in number]?;
 };
 
 type Type = {
   // comment
-  readonly [T in number];
+  readonly [T in number]?;
 };
 
 type Type = {
   // foo
-  /* bar */ readonly [T in number];
+  /* bar */ readonly [T in number]?;
 };
