===================================================================
--- theirs.js
+++ ours.js
@@ -18,13 +18,13 @@
 
 (a?.b).c();
 (a?.b[c]).c();
 
-a?.b?.c.d?.e;
+(a?.b)?.c.d?.e;
 (a ? b : c)?.d;
 
 (list || list2)?.length;
-(list || list2)?.[list || list2];
+(list || list2)?.[(list || list2)];
 
 async function HelloWorld() {
   var x = (await foo.bar.blah)?.hi;
   a?.[await b];
@@ -35,19 +35,19 @@
 a?.[b?.c].d();
 a[b?.c]?.d();
 a?.[b?.c]?.d();
 
-one?.fn();
+(one?.fn());
 (one?.two).fn();
 (one?.two)();
 (one?.two())();
-one.two?.fn();
+(one.two?.fn());
 (one.two?.three).fn();
-one.two?.three?.fn();
+(one.two?.three?.fn());
 
-one?.();
+(one?.());
 (one?.())();
-one?.()?.();
+(one?.())?.();
 
 (one?.()).two;
 
 a?.[b ? c : d];
@@ -61,27 +61,27 @@
 (() => f)?.x;
 (a?.(x)).x;
 (
   aaaaaaaaaaaaaaaaaaaaaaaa &&
-  aaaaaaaaaaaaaaaaaaaaaaaa &&
-  aaaaaaaaaaaaaaaaaaaaaaaa
+    aaaaaaaaaaaaaaaaaaaaaaaa &&
+    aaaaaaaaaaaaaaaaaaaaaaaa
 )?.();
 
-let f = () => ({})?.();
-let g = () => ({})?.b;
-a = () => ({})?.() && a;
-a = () => ({})?.()() && a;
-a = () => ({})?.().b && a;
-a = () => ({})?.b && a;
-a = () => ({})?.b() && a;
-(a) => ({})?.()?.b && 0;
-(a) => ({})?.b?.b && 0;
-(x) => ({})?.()();
-(x) => ({})?.().b;
-(x) => ({})?.b();
-(x) => ({})?.b.b;
-({})?.a().b();
-({ a: 1 })?.entries();
+let f = () => ({}?.());
+let g = () => ({}?.b);
+a = () => ({}?.() && a);
+a = () => ({}?.()() && a);
+a = () => ({}?.().b && a);
+a = () => ({}?.b && a);
+a = () => ({}?.b() && a);
+(a) => ({}?.()?.b && 0);
+(a) => ({}?.b?.b && 0);
+(x) => ({}?.()());
+(x) => ({}?.().b);
+(x) => ({}?.b());
+(x) => ({}?.b.b);
+({}?.a().b());
+({ a: 1 }?.entries());
 
 new (foo?.bar)();
 new (foo?.bar())();
 new (foo?.())();
