===================================================================
--- theirs.js
+++ ours.js
@@ -6,12 +6,14 @@
     console.log(num);
   }
 }
 
-(async () => {
-  for await (num of asyncIterable) {
-    console.log(num);
+(
+  async () => {
+    for await (num of asyncIterable) {
+      console.log(num);
+    }
+    for await (const x of delegate_yield()) {
+      x;
+    }
   }
-  for await (const x of delegate_yield()) {
-    x;
-  }
-})();
+)();
