===================================================================
--- theirs.js
+++ ours.js
@@ -26,51 +26,57 @@
 );
 
 function render() {
   return (
-    <View>
-      <Image
-        onProgress={(e) =>
-          this.setState({
-            progress: Math.round(
-              (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
-            ),
-          })
-        }
-      />
-    </View>
+    (
+      <View>
+        <Image
+          onProgress={(e) =>
+            this.setState({
+              progress: Math.round(
+                (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
+              ),
+            })
+          }
+        />
+      </View>
+    )
   );
 }
 
 function render() {
   return (
-    <View>
-      <Image
-        onProgress={(e) =>
-          this.setState({
-            progress: Math.round(
-              (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
-            ),
-          })
-        }
-      />
-    </View>
+    (
+      <View>
+        <Image
+          onProgress={(e) =>
+            this.setState({
+              progress: Math.round(
+                (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
+              ),
+            })
+          }
+        />
+      </View>
+    )
   );
 }
 
 function render() {
   return (
-    <View>
-      <Image
-        onProgress={(e) =>
-          this.setState({
-            progress: Math.round(
-              (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
-            ),
-          })
-        }
-      />
-    </View>
+    (
+      <View>
+        <Image
+          onProgress={(e) =>
+            this.setState({
+              progress: Math.round(
+                (100 * e.nativeEvent.loaded) / e.nativeEvent.total,
+              ),
+            })
+          }
+        />
+      </View>
+    )
   );
 }
 
 jest.mock(
@@ -345,13 +351,15 @@
 );
 
 foo(
   (
-    a = (({
-      a,
+    a = (
+      ({
+        a,
 
-      b,
-    }) => {})(),
+        b,
+      }) => {}
+    )(),
   ) => {},
 );
 
 foo(
