===================================================================
--- theirs.js
+++ ours.js
@@ -11,8 +11,8 @@
                 justify-content: center;
                 text-align: center;`
       : ""}
   @media (max-width: ${(props) =>
-    props.noBreakPoint ? "0" : constants.layout.breakpoint.break1}px) {
+    (props.noBreakPoint ? "0" : constants.layout.breakpoint.break1)}px) {
     font-size: 2em;
   }
 `;
