===================================================================
--- theirs.js
+++ ours.js
@@ -1,9 +1,10 @@
 a =
-  // Comment 1
-  Math.random() * (yRange * (1 - minVerticalFraction)) +
-  minVerticalFraction * yRange -
-  offset;
+  (
+    // Comment 1
+    (Math.random() * (yRange * (1 - minVerticalFraction))) +
+      (minVerticalFraction * yRange)
+  ) - offset;
 
 a +
   a +
   a + // comment
@@ -43,17 +44,17 @@
 ];
 
 !(
   a +
-  a + // comment
-  a +
-  !(
-    b +
-    b +
-    b + // comment
-    b +
-    b
-  )
+    a + // comment
+    a +
+    !(
+      b +
+        b +
+        b + // comment
+        b +
+        b
+    )
 );
 
 const logical_expression =
   aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa /* trailing comment */ &&
