===================================================================
--- theirs.js
+++ ours.js
@@ -1,7 +1,9 @@
-((b) => (c) => (d) => {
-  return 3;
-})(x);
+(
+  (b) => (c) => (d) => {
+    return 3;
+  }
+)(x);
 
 function f(
   a = (fooLorem) => (bazIpsum) => (barLorem) => {
     return 3;
@@ -9,17 +11,16 @@
 ) {}
 
 (
   (fooLoremIpsumFactory) =>
-  (bazLoremIpsumFactory) =>
-  (barLoremIpsumServiceFactory) => {
-    return 3;
-  }
+    (bazLoremIpsumFactory) =>
+    (barLoremIpsumServiceFactory) => {
+      return 3;
+    }
 )(x);
 
 (
-  (b) => (c) => (d) =>
-    b + fooLoremIpsumFactory(c) - bazLoremIpsumFactory(b + d)
+  (b) => (c) => (d) => b + fooLoremIpsumFactory(c) - bazLoremIpsumFactory(b + d)
 )(x, fooLoremIpsumFactory, fooLoremIpsumFactory);
 
 (
   (fooLorem) => (bazIpsum) => (barLorem) =>
@@ -27,8 +28,8 @@
 )(boo);
 
 (
   (fooLoremIpsumFactory) =>
-  (bazLoremIpsumFactory) =>
-  (barLoremIpsumServiceFactory) =>
-    b + fooLoremIpsumFactory(c) - bazLoremIpsumFactory(b + d)
+    (bazLoremIpsumFactory) =>
+    (barLoremIpsumServiceFactory) =>
+      b + fooLoremIpsumFactory(c) - bazLoremIpsumFactory(b + d)
 )(x);
