===================================================================
--- theirs.js
+++ ours.js
@@ -38,10 +38,11 @@
   console.log("Followers:", followers),
 );
 // Followers: ["STEVE","SUZY"]
 
-const mapStateToProps = (state) => ({
-  users: R.compose(
-    R.filter(R.propEq("status", "active")),
-    R.values,
-  )(state.users),
-});
+const mapStateToProps = (state) =>
+  ({
+    users: R.compose(
+      R.filter(R.propEq("status", "active")),
+      R.values,
+    )(state.users),
+  });
