===================================================================
--- theirs.jsx
+++ ours.jsx
@@ -1,13 +1,15 @@
 const tabs = [
   {
     title: "General Info",
     content: (
-      <GeneralForm
-        long-attribute="i-need-long-value-here"
-        onSave={onSave}
-        onCancel={onCancel}
-        countries={countries}
-      />
+      (
+        <GeneralForm
+          long-attribute="i-need-long-value-here"
+          onSave={onSave}
+          onCancel={onCancel}
+          countries={countries}
+        />
+      )
     ),
   },
 ];
