===================================================================
--- theirs.jsx
+++ ours.jsx
@@ -38,20 +38,24 @@
   </div>
 );
 
 regression_extra_newline = (
-  <div>
-    <span className="nuclide-console-new-messages-notification-icon icon icon-nuclicon-arrow-down" />
-    New Messages
-  </div>
+  (
+    <div>
+      <span className="nuclide-console-new-messages-notification-icon icon icon-nuclicon-arrow-down" />
+      New Messages
+    </div>
+  )
 );
 
 regression_extra_newline_2 = (
-  <div>
-    (
-    <FormattedMessage
-      id="some-id"
-      defaultMessage="some loooooooooooooooooooooooooooong default"
-    />
-    )
-  </div>
+  (
+    <div>
+      (
+      <FormattedMessage
+        id="some-id"
+        defaultMessage="some loooooooooooooooooooooooooooong default"
+      />
+      )
+    </div>
+  )
 );
