===================================================================
--- theirs.jsx
+++ ours.jsx
@@ -1,9 +1,12 @@
-const render = (items) => (
-  <div>
-    {
-      /* eslint-disable */
-      items.map((item) => null)
-      /* eslint-enable */
-    }
-  </div>
-);
+const render = (items) =>
+  (
+    (
+      <div>
+        {
+          /* eslint-disable */
+          items.map((item) => null)
+          /* eslint-enable */
+        }
+      </div>
+    )
+  );
