===================================================================
--- theirs.js
+++ ours.js
@@ -62,23 +62,17 @@
 );
 
 func(
   () =>
-    (
-      // comment
-      a, b, c
-    ),
+    // comment
+    (a, b, c),
 );
 func(
   () => () =>
-    (
-      // comment
-      a, b, c
-    ),
+    // comment
+    (a, b, c),
 );
 func(
   () => () => () =>
-    (
-      // comment
-      a, b, c
-    ),
+    // comment
+    (a, b, c),
 );
