===================================================================
--- theirs.js
+++ ours.js
@@ -9,9 +9,9 @@
  * @template T
  * @param {Type} type
  * @param {T} value
  * @return {Value}
- *//**
+ */ /**
  * @param {Type} type
  * @return {Value}
  */
 function value(type, value) {
@@ -22,7 +22,7 @@
   }
 }
 
 /** Trailing nestled comment 1
- *//** Trailing nestled comment 2
- *//** Trailing nestled comment 3
+ */ /** Trailing nestled comment 2
+ */ /** Trailing nestled comment 3
  */
