===================================================================
--- theirs.ts
+++ ours.ts
@@ -2,12 +2,9 @@
 
 module A {
   export var x = "hello world";
   export class Point {
-    constructor(
-      public x: number,
-      public y: number,
-    ) {}
+    constructor(x: number, y: number) {}
   }
   export module B {
     export interface Id {
       name: string;
@@ -30,12 +27,9 @@
   }
 
   export module Y {
     export class Point {
-      constructor(
-        public x: number,
-        public y: number,
-      ) {}
+      constructor(x: number, y: number) {}
     }
   }
 }
 
@@ -48,9 +42,9 @@
 var n: { x: number; y: number } = new Z.y.Point(0, 0);
 
 module K {
   export class L {
-    constructor(public name: string) {}
+    constructor(name: string) {}
   }
 
   export module L {
     export var y = 12;
