===================================================================
--- theirs.js
+++ ours.js
@@ -11,25 +11,31 @@
       ? value3
       : value4;
 
 const StorybookLoader = ({ match }) =>
-  match.params.storyId === "button" ? (
-    <ButtonStorybook />
-  ) : match.params.storyId === "color" ? (
-    <ColorBook />
-  ) : match.params.storyId === "typography" ? (
-    <TypographyBook />
-  ) : match.params.storyId === "loading" ? (
-    <LoaderStorybook />
-  ) : match.params.storyId === "deal-list" ? (
-    <DealListStory />
-  ) : (
-    <Message>
-      <Title>{"Missing story book"}</Title>
-      <Content>
-        <BackButton />
-      </Content>
-    </Message>
+  (
+    match.params.storyId === "button" ? (
+      <ButtonStorybook />
+    ) : match.params.storyId === "color" ? (
+      <ColorBook />
+    ) : match.params.storyId === "typography" ? (
+      <TypographyBook />
+    ) : match.params.storyId === "loading" ? (
+      <LoaderStorybook />
+    ) : match.params.storyId === "deal-list" ? (
+      <DealListStory />
+    ) : (
+      (
+        (
+          <Message>
+            <Title>{"Missing story book"}</Title>
+            <Content>
+              <BackButton />
+            </Content>
+          </Message>
+        )
+      )
+    )
   );
 
 const message =
   i % 3 === 0 && i % 5 === 0
@@ -65,26 +71,26 @@
 const foo = (
   <div
     className={
       "match-achievement-medal-type type" +
-      (medals[0].record
-        ? "-record"
-        : medals[0].unique
-          ? "-unique"
-          : medals[0].type)
+      (
+        medals[0].record
+          ? "-record"
+          : (medals[0].unique ? "-unique" : medals[0].type)
+      )
     }
   >
     {medals[0].record
-      ? i18n("Record")
+      ? (i18n("Record"))
       : medals[0].unique
-        ? i18n("Unique")
+        ? (i18n("Unique"))
         : medals[0].type === 0
-          ? i18n("Silver")
+          ? (i18n("Silver"))
           : medals[0].type === 1
-            ? i18n("Gold")
+            ? (i18n("Gold"))
             : medals[0].type === 2
-              ? i18n("Platinum")
-              : i18n("Theme")}
+              ? (i18n("Platinum"))
+              : (i18n("Theme"))}
   </div>
 );
 
 a
@@ -98,25 +104,27 @@
 const config = {
   onFailure:
     onFailure !== undefined
       ? onFailure
-      : (error) => {
-          notify(
-            typeof error === "string"
-              ? error
-              : error.message || "ra.notification.http_error",
-            "warning",
-            {
-              _:
-                typeof error === "string"
-                  ? error
-                  : error && error.message
-                    ? error.message
-                    : undefined,
-            },
-          );
-          refresh();
-        },
+      : (
+          (error) => {
+            notify(
+              typeof error === "string"
+                ? error
+                : error.message || "ra.notification.http_error",
+              "warning",
+              {
+                _:
+                  typeof error === "string"
+                    ? error
+                    : error && error.message
+                      ? error.message
+                      : undefined,
+              },
+            );
+            refresh();
+          }
+        ),
 };
 
 showNotification(
   typeof error === "string" ? error : error.message || body,
