===================================================================
--- theirs.ts
+++ ours.ts
@@ -1,10 +1,7 @@
 class C {
   y: string;
-  constructor(
-    private x: string,
-    protected z: string,
-  ) {}
+  constructor(x: string, z: string) {}
 }
 
 var c: C;
 var r = c.y;
@@ -12,13 +9,9 @@
 var r3 = c.z; // error
 
 class D<T> {
   y: T;
-  constructor(
-    a: T,
-    private x: T,
-    protected z: T,
-  ) {}
+  constructor(a: T, x: T, z: T) {}
 }
 
 var d: D<string>;
 var r = d.y;
