===================================================================
--- theirs.js
+++ ours.js
@@ -1,8 +1,10 @@
 function jsx() {
   return (
-    // Comment
-    <div />
+    (
+      // Comment
+      <div />
+    )
   );
 }
 
 function unary() {
@@ -17,82 +19,100 @@
 }
 
 function logical() {
   return (
-    // Reason for 42
-    42 && 84
+    (
+      // Reason for 42
+      42
+    ) && 84
   );
 }
 
 function binary() {
   return (
-    // Reason for 42
-    42 * 84
+    (
+      // Reason for 42
+      42
+    ) * 84
   );
 }
 
 function binaryInBinaryLeft() {
   return (
-    // Reason for 42
-    42 *
+    (
+      // Reason for 42
+      42
+    ) *
       84 +
     2
   );
 }
 
 function binaryInBinaryRight() {
   return (
-    // Reason for 42
-    42 +
+    (
+      // Reason for 42
+      42
+    ) +
     84 * 2
   );
 }
 
 function conditional() {
   return (
-    // Reason for 42
-    42
+    (
+      // Reason for 42
+      42
+    )
       ? 1
       : 2
   );
 }
 
 function binaryInConditional() {
   return (
-    // Reason for 42
-    42 * 3
+    (
+      // Reason for 42
+      42
+    ) * 3
       ? 1
       : 2
   );
 }
 
 function call() {
   return (
-    // Reason for a
-    a()
+    (
+      // Reason for a
+      a
+    )()
   );
 }
 
 function memberInside() {
   return (
-    // Reason for a.b
-    a.b.c
+    (
+      // Reason for a.b
+      a.b
+    ).c
   );
 }
 
 function memberOutside() {
   return (
-    // Reason for a
-    a.b.c
+    (
+      // Reason for a
+      a
+    ).b.c
   );
 }
 
 function memberInAndOutWithCalls() {
   return (
-    aFunction
-      .b// Reason for a
-      ()
-      .c.d()
+    (
+      // Reason for a
+      aFunction.b()
+    ).c.d()
   );
 }
 
 function excessiveEverything() {
@@ -118,15 +138,17 @@
 }
 
 function taggedTemplate() {
   return (
-    // Reason for a
-    a`b`
+    (
+      // Reason for a
+      a
+    )`b`
   );
 }
 
 function inlineComment() {
-  return /* hi */ 42 || 42;
+  return (/* hi */ 42) || 42;
 }
 
 function multilineBlockSameLine() {
   return (
@@ -146,25 +168,29 @@
 }
 
 function multilineBlockSameLineJsx() {
   return (
-    /**
-     * JSX Same line
-     */ <div></div>
+    (
+      /**
+       * JSX Same line
+       */ <div></div>
+    )
   );
 }
 
 function multilineBlockNextLineJsx() {
   return (
-    /**
-     * JSX Next line
-     */
-    <div></div>
+    (
+      /**
+       * JSX Next line
+       */
+      <div></div>
+    )
   );
 }
 
 function singleLineBlockSameLine() {
-  return /** Result -> */ "result";
+  return (/** Result -> */ "result");
 }
 
 function singleLineBlockNextLine() {
   return (
