===================================================================
--- theirs.js
+++ ours.js
@@ -14,9 +14,9 @@
     RecordImport.find({}, "", {}, (err, batches) => {
       callback(
         null,
         batches.filter(
-          (batch) => batch.state !== "error" && batch.state !== "completed",
+          (batch) => (batch.state !== "error" && batch.state !== "completed"),
         ),
       );
     });
   };
