===================================================================
--- theirs.jsx
+++ ours.jsx
@@ -54,51 +54,63 @@
   </fbt>
 );
 
 x = (
-  <fbt>
-    {hour}:{minute}:{second}
-  </fbt>
+  (
+    <fbt>
+      {hour}:{minute}:{second}
+    </fbt>
+  )
 );
 
 x = (
-  <fbt>
-    {hour}
-    :
-    {minute}
-    :
-    {second}
-  </fbt>
+  (
+    <fbt>
+      {hour}
+      :
+      {minute}
+      :
+      {second}
+    </fbt>
+  )
 );
 
 x = (
-  <fbt>
-    {hour}:
-    {minute}:
-    {second}
-  </fbt>
+  (
+    <fbt>
+      {hour}:
+      {minute}:
+      {second}
+    </fbt>
+  )
 );
 
 first = (
-  <fbt>
-    Text<br />
-    More text<br />
-    And more<br />
-  </fbt>
+  (
+    <fbt>
+      Text<br />
+      More text<br />
+      And more<br />
+    </fbt>
+  )
 );
 
 second = (
-  <fbt>
-    Text<br />More text<br />And more<br />
-  </fbt>
+  (
+    <fbt>
+      Text<br />More text<br />And more<br />
+    </fbt>
+  )
 );
 
 third = (
-  <fbt>
-    Text
-    <br />
-    More text
-    <br />
-    And more
-    <br />
-  </fbt>
+  (
+    <fbt>
+      Text
+      <br />
+      More text
+      <br />
+      And more
+      <br />
+    </fbt>
+  )
 );
