===================================================================
--- theirs.jsx
+++ ours.jsx
@@ -1,13 +1,17 @@
 function HelloWorld() {
   return (
-    <div>
+    (
       <div>
-        foo
-        <br />
-        bar{" "}
-        <span className="font-semibold">foobar foobar foobar foobar 12345</span>
-        , foobar foobar foobar
+        <div>
+          foo
+          <br />
+          bar{" "}
+          <span className="font-semibold">
+            foobar foobar foobar foobar 12345
+          </span>
+          , foobar foobar foobar
+        </div>
       </div>
-    </div>
+    )
   );
 }
